ts/Extensions/Annotations/Popup.ts

```diff
@@ -307,55 +346,79 @@ H.Popup.prototype = {
             lhsCol;
 
         // left column
-        lhsCol = createElement(DIV, {
-            className: PREFIX + 'popup-lhs-col'
-        }, null as any, container);
+        lhsCol = createElement(
+            DIV,
+            {
+                className: PREFIX + 'popup-lhs-col'
+            },
+            void 0,
+            container
+        );
 
         // right column
-        rhsCol = createElement(DIV, {
-            className: PREFIX + 'popup-rhs-col'
-        }, null as any, container);
+        rhsCol = createElement(
+            DIV,
+            {
+                className: PREFIX + 'popup-rhs-col'
+            },
+            void 0,
+            container
+        );
 
         // wrapper content
-        createElement(DIV, {
-            className: PREFIX + 'popup-rhs-col-wrapper'
-        }, null as any, rhsCol);
+        createElement(
+            DIV,
+            {
+                className: PREFIX + 'popup-rhs-col-wrapper'
+            },
+            void 0,
+            rhsCol
+        );
 
         return {
             lhsCol: lhsCol,
             rhsCol: rhsCol
         };
     },
+
     /**
      * Create input with label.
+     *
      * @private
+     *
      * @param {string} option
-     * Chain of fields i.e params.styles.fontSize
-     * @param {string} type
-     * Indicator type
-     * @param {Highhcharts.HTMLDOMElement}
-     * Container where elements should be added
-     * @param {string} value
-     * Default value of input i.e period value is 14, extracted from
-     * defaultOptions (ADD mode) or series options (EDIT mode)
+     *        Chain of fields i.e params.styles.fontSize separeted by the dot.
+     *
+     * @param {string} indicatorType
+     *        Type of the indicator i.e. sma, ema...
+     *
+     * @param {HTMLDOMElement} parentDiv
+     *        HTML parent element.
+     *
+     * @param {Highcharts.InputAttributes} inputAttributes
+     *        Attributes of the input.
+     *
+     * @return {HTMLSelectElement}
```

Comment:
Here is a type mismatch and a description missing. Without one does not now, which element to expect (parent, label or input).